reservation_units/utils/tests/test_reservation_unit_state_helper.py

```diff
@@ -0,0 +1,80 @@
+import datetime
+
+from assertpy import assert_that
+from django.test.testcases import TestCase
+
+from reservation_units.enums import ReservationUnitState
+from reservation_units.models import ReservationUnit
+from reservation_units.tests.factories import ReservationUnitFactory
+from reservation_units.utils.reservation_unit_state_helper import get_state
+
+
+class ReservationUnitStateHelperTestCase(TestCase):
```

Comment:
Adding a test for raising the `ValueError` would be nice.

Reply:
`ValueError` is now gone ☺️